src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar calendar--month-length--29 calendar--start-day--fri  calendar--pos">
```

Comment:
you can try to change the tag 'main' to 'div' it should solve the problem with test